Process/Voting/README.md

```diff
@@ -0,0 +1,7 @@
+# Voting
+
+## [Double Democracy](Double-Democracy/)
+
+A procedure for combining both corrective and constructive democracy
+
+
```

Comment:
TBD add rules for voting with github pull requests and rules for voting on multiple options